packages/core/src/shader/utils/testContext.ts

```diff
@@ -0,0 +1,61 @@
+import { settings } from '../../settings';
```

Comment:
It's not always easy to tell from GitHub, but can you make sure you do a `git mv` on **getTextContext.ts** to **testContext.ts**? Creating a new file destroys the git history, which is bad for blaming.

Reply:
Sure. Types are not generated correctly, so it is not ready to merge anyways.